### PR TITLE
updated examples for 1.0.2 release

### PR DESCRIPTION
**Pull request by druspencer**
- Updated examples for the `1.0.2` release of the client
- Updates for Monaco Protocol nomenclature changes
  - `back/lay` >> `for/against`
  - `betOrder` >> `order`
- Brings in examples for getting market outcomes and market outcome names
- Brings in examples for getting new trade accounts
- Adds a secondary wallet to aid matching orders (with scripts to create for and against orders that will match)
- Updates readme to reflect new wallet

### CODE REVIEW

**Review by peter-betdex — APPROVED** [review]

File: examples/cli/src/get_market_outcome_titles.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { getMarketOutcomeTitlesByMarket } from "@monaco-protocol/client";
+import { PublicKey } from "@solana/web3.js";
+import { getProgram, logJson, log } from "./utils";
+
+async function marketOutcomeTitles(marketPk: PublicKey){
```
Comment (pawel-betdex) [inline examples/cli/src/get_market_outcome_titles.ts:5]: no get?

File: examples/cli/src/get_market_outcomes.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { getMarketOutcomesByMarket } from "@monaco-protocol/client";
+import { PublicKey } from "@solana/web3.js";
+import { getProgram, logJson, log } from "./utils";
+
+async function marketOutcomes(marketPk: PublicKey){
```
Comment (pawel-betdex) [inline examples/cli/src/get_market_outcomes.ts:5]: no get?

Comment (eoin-betdex) [reply examples/cli/src/get_market_outcomes.ts:5]: I think these are just example functions, they're not actually returning the data, just logging it.

Comment (druspencer) [reply examples/cli/src/get_market_outcomes.ts:5]: yeah, just trying to keep them succinct and distinct from the function they are calling `getMarketOutcomesByMarket`

**Review by pawel-betdex — APPROVED** [review]

**Pull request merged**